File: src/main/java/com/rednorte/sigle/listas_service/model/Paciente.java
package com.rednorte.sigle.listas_service.model;

import jakarta.persistence.*;
import jakarta.validation.constraints.Email;
import jakarta.validation.constraints.NotBlank;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDate;

@Entity
@Table(name = "pacientes")
@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class Paciente {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    @Column(unique = true, nullable = false)
    @NotBlank(message = "El RUT es obligatorio")
    private String rut;

    @NotBlank(message = "El nombre es obligatorio")
    private String nombre;

    private String apellido;

    @Column(name = "fecha_nacimiento")
    private LocalDate fechaNacimiento;

    @Email(message = "El email no es válido")
    private String email;

    private String telefono;

    @Column(name = "establecimiento_id")
    private Long establecimientoId;
}

File: src/models/Paciente.js
const { DataTypes } = require('sequelize');
const sequelize = require('../config/database');

const Paciente = sequelize.define('Paciente', {
  id: { type: DataTypes.BIGINT, autoIncrement: true, primaryKey: true },
  rut: { type: DataTypes.STRING, allowNull: false, unique: true },
  nombre: { type: DataTypes.STRING, allowNull: false },
  apellido: { type: DataTypes.STRING },
  fechaNacimiento: { type: DataTypes.DATEONLY, field: 'fecha_nacimiento' },
  email: { type: DataTypes.STRING },
  telefono: { type: DataTypes.STRING },
  establecimientoId: { type: DataTypes.BIGINT, field: 'establecimiento_id' },
}, { tableName: 'pacientes', timestamps: false });

module.exports = Paciente;
